source/pidomus.cc

```diff
@@ -887,24 +899,45 @@ piDoMUS<dim, spacedim, LAC>::solve_jacobian_system(const double /*t*/,
     }
   else
     {
+      unsigned int iterations = 0;
+      try
+        {
+          PrimitiveVectorMemory<typename LAC::VectorType> mem;
+          SolverControl solver_control (matrices[0]->m(),
+                                        jacobian_solver_tolerance);
+          SolverFGMRES<typename LAC::VectorType>
+          solver(solver_control, mem,
+                 typename SolverFGMRES<typename LAC::VectorType>::AdditionalData(30, true));
```

Comment:
per il momento ho messo 30 anziché 50 qua, e la possibilità di modulare in maniera diversa la tolleranza.